### PR TITLE
New cell types - olfactory

### PR DESCRIPTION
**Pull request by UlrikeS91**
from https://github.com/openMetadataInitiative/openMINDS_controlledTerms/pull/459

I tried to incorporate @lzehl feedback as good as possible.

### CODE REVIEW

**Comment by UlrikeS91** [comment]
Please wait with the review until after I revised this again. I just found https://github.com/openMetadataInitiative/openMINDS_controlledTerms/issues/240 and will add the missing ones for the accessory olf. bulb and revise the existing ones.

**Comment by UlrikeS91** [comment]
I've updated the new cell types to be the ones from the main olfactory bulb. Since @apdavison said that there were no use cases for the accessory olfactory bulb, I would like to wait with those until after we did the clean-up for the cell types. @lzehl could you please review the PR?

File: instances/latest/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbGranuleNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "'Main olfactory bulb granule neurons' are axonless inhibitory interneurons and form the majority of neurons in the vertebrate main olfactory bulb [adapted from [Egger et al. (2003)](https://doi.org/10.1523%2FJNEUROSCI.23-20-07551.2003)].",
```
Comment (lzehl) [inline instances/latest/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld:7]: new convention? "Axonless inhibitory interneurons that form the majority of neurons in the vertebrate main olfactory bulb ..."

Comment (lzehl) [reply instances/latest/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld:7]: OR "A subclass of neurons, axonless and inhibitory, situated in the  the main olfactory bulb of vertebrates."

File: instances/latest/terminologies/cellType/mainOlfactoryBulbMitralNeuron.jsonld [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbMitralNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "'Mitral neurons' are a neuronal cell type in the mammalian olfactory bulb, distinguished by the position of their somata located in an orderly row in the mitral cell layer of the bulb. [from [Wikipedia](https://en.wikipedia.org/wiki/Mitral_cell#Structure)]",
```
Comment (lzehl) [inline instances/latest/terminologies/cellType/mainOlfactoryBulbMitralNeuron.jsonld:7]: adapt to new convention?

File: instances/latest/terminologies/cellType/mainOlfactoryBulbPeriglomerularNeuron.jsonld [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbPeriglomerularNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "'Periglomerular neurons' are small intrinsic neurons in the glomerular layer of the main olfactory bulb, with cell bodies surrounding the olfactory glomerulus. [adapted from [InterLex](http://uri.interlex.org/base/ilx_0107934)]",
```
Comment (lzehl) [inline instances/latest/terminologies/cellType/mainOlfactoryBulbPeriglomerularNeuron.jsonld:7]: adapt to new convention?

File: instances/latest/terminologies/cellType/mainOlfactoryBulbTuftedNeuron.jsonld [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbTuftedNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "'Tufted cells' are principle neurons of the mammalian main olfactory bulb.",
```
Comment (lzehl) [inline instances/latest/terminologies/cellType/mainOlfactoryBulbTuftedNeuron.jsonld:7]: adapt to new convention? if not, correct to full name of cell

**Review by lzehl — CHANGES_REQUESTED** [review]
only remove name from definition (and make clear if it is a superclass)

**Review by lzehl — APPROVED** [review]
updated namespace, updated some definitions, populated across versions @Raphael-Gazzotti could you double check and merge when okay ?

**Review by lzehl — APPROVED** [review]

File: instances/v3.0/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/cellType/mainOlfactoryBulbGranuleNeuron",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/CellType",
+  "definition": "Axonless inhibitory interneurons and form the majority of neurons in the vertebrate main olfactory bulb [adapted from [Egger et al. (2003)](https://doi.org/10.1523%2FJNEUROSCI.23-20-07551.2003)].",
```
[the remark this answers is not in the document]
Comment (lzehl) [reply instances/v3.0/terminologies/cellType/mainOlfactoryBulbGranuleNeuron.jsonld:7]: Done

**Pull request merged**